src/modules/BlackLagerModule.cpp

```diff
@@ -1,26 +1,25 @@
 #include "BlackLagerModule.h"
-#include "MeshService.h"
+#include "NodeDB.h"
+#include "PowerFSM.h"
 #include "configuration.h"
-#include "main.h"
-
-#include <assert.h>
 
 BlackLagerModule *blackLagerModule;
 
-MeshPacket *BlackLagerModule::allocReply()
+/**
+ * Text messaging module with digital signatures.
+ */
+ProcessMessage BlackLagerModule::handleReceived(const MeshPacket &mp)
 {
-    assert(currentRequest); // should always be !NULL
```

Comment:
Is it certain we do not need to assert the currentRequest anymore?

Reply:
Replaced the assert with a debug log message so an error can be logged instead of the function automatically returning if the assert fails. We can add asserts later when writing test cases.